packages/appsync-emulator-serverless/httpSource.js

```diff
@@ -1,7 +1,11 @@
 const fetch = require('node-fetch');
 
 const httpSource = async (endpoint, { resourcePath, method, params }) => {
-  const response = await fetch(endpoint + resourcePath, {
+  const { query } = params || {};
+  const queryPath =
+    query === undefined ? '' : `?${new URLSearchParams(Object.entries(query))}`;
```

Comment:
curious why you wrapped `query` with Object.entries.  [Here](https://nodejs.org/api/url.html#url_class_urlsearchparams) it says URLSearchParams accepts an object.

Reply:
Good call. Just committed an update to remove that.